radixdlt-core/radixdlt/src/test/java/com/radixdlt/network/messaging/InboundMessageTest.java

```diff
@@ -83,7 +83,7 @@
 
 	@Before
 	public void setUp() {
-		this.inboundMessage = InboundMessage.of(nodeId, this.message);
+		this.inboundMessage = new InboundMessage(0L, nodeId, this.message);
```

Comment:
What is the reason for switching from factory method to direct constructor call?

Reply:
I'm not sure if it makes sense to keep it since we can't make record's constructor private anyway. wdyt?